src/open_dread_rando/pickups/pickup.py

```diff
@@ -113,8 +113,13 @@ def patch_multi_item_pickup(self, bmsad: dict) -> dict:
         pickable: dict = bmsad["property"]["components"]["PICKABLE"]
         script: dict = bmsad["property"]["components"]["SCRIPT"]
 
+        item_id = "ITEM_NONE"
+        first_progression = self.pickup["resources"][0][0]["item_id"]
+        if first_progression in {"ITEM_WEAPON_WIDE_BEAM", "ITEM_WEAPON_SUPER_MISSILE"}:
+            item_id = first_progression
+
         set_custom_params: dict = pickable["functions"][0]["params"]
-        set_custom_params["Param1"]["value"] = "ITEM_NONE"
+        set_custom_params["Param1"]["value"] = item_id
```

Comment:
Can't follow: What is that about? We used `ITEM_NONE` because we are giving the stuff via lua, why do we need the first progression here?

Reply:
weird stuff. I should probably add a comment... and check whether it's still necessary

Reply:
Hmmm I read the comment.
If that's really needed, Bosses and multiworld are doing lua only, and have the potential to be broken again for these specific pickups.

Can you check that?

Reply:
Corpius granted Ice Missiles correctly in my testing. Unsure about multi, but I figure we'll find out after it's merged lol